### ISSUE TITLE
Travis CI data access

### ISSUE THREAD

**Issue by Stefan-Endres** [Stefan-Endres/DWPM-Mixture-Model#38]
The build unittests are failing because the config.cfg file is not in the public GitHub repository, when a customized one is added it cannot find the correct directory where the data is stored locally. 

Is there anyway to fix this? Maybe by adding a remote data storage for the .csv files?

**Comment by alchemyst** [comment]
The real issue is that the tests shouldn't require the data. The good way to handle this would be to use `mock` to build an object which does the same things as the data object but for the (hopefully) smaller cases that the tests test. These difficulties with testing the routines in isolation indicate design problems, and we should avoid just plastering over them.

**Comment by Stefan-Endres** [comment]
I agree. I will keep this issue open until we deal with fixing the data handling entirely.
